vxquery-core/src/test/java/org/apache/vxquery/datamodel/AbstractPointableTest.java

```diff
@@ -28,7 +28,9 @@
 
     protected void getTaggedValuePointable(Object value, IPointable result) throws IOException {
```

Comment:
The objectbuilder removes the type tag, but requires a UTF8StringPointable to be passed in, so it wouldn't be necessary to have a pointable without the tag when testing